Yearn-agents/PPSValues/src/agent.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { HandleBlock } from "forta-agent";
+import { TestBlockEvent } from "forta-agent-tools";
+import agent, { createFinding } from "./agent";
+import MockWeb3, { mockPrice } from "./mock";
+
+describe("PPS ( Price per share ) agent", () => {
+  let handleBlock: HandleBlock;
+
+  beforeAll(() => {
+    let mockWeb3 = {
+      eth: {
+        Contract: MockWeb3.build_Mock(),
+      },
+    } as any;
+    handleBlock = agent.provideHandleFunction(mockWeb3 as any);
+  });
+
+  it("returns empty findings if pps increases", async () => {
+    const blockEvent = new TestBlockEvent();
+    mockPrice.mockReturnValueOnce(1.1);
```

Comment:
I would like to suggest you to not use `1.1` or float numbers at all, because `pps` returns an `uint256`

Reply:
Actually no. If you look at the code :
` return balance().mul(1e18).div(totalSupply());` 
Wad math is used, so any number will do.

Reply:
Just to make my point clear, this is the `pps` function signature
```python
def pricePerShare() -> uint256:
```
you are mocking the value returned by that function